.ci/release.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+## This script runs the release given the different environment variables
```

Comment:
This particular script was initially defined in JJB. It has been adapted to support running on Buildkite, hence there are some specific echo and env variables in place to ensure the logs UI can be collapsed and no secrets are printed in the logs.